my_game.py

```diff
@@ -629,6 +668,7 @@ def main():
     """
     # Create a window to hold views
     window = arcade.Window(SCREEN_WIDTH, SCREEN_HEIGHT)
+    arcade.Window.center_window(window)
```

Comment:
What is being done here?

Reply:
That's just so the screen centers, I'll remove it, it's just so it's easier for me to see in the terminal at the same time, without having to move the screen each and every time